companies/psd-bb.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "psd-bb",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "PSD Bank Berlin-Brandenburg eG",
+    "address": "Datenschutz\nHandjerystr. 34–36\n12159 Berlin\nDeutschland",
```

Comment:
Please don't add the ATTN line to the address, as these are added automatically by our website ([see](https://github.com/datenanfragen/data#addresses)).